dietpi/dietpi-drive_manager

```diff
@@ -124,31 +120,25 @@
 
 			init=0
 
-			G_DIETPI-NOTIFY 2 "Detecting drives, please wait..."
+			G_DIETPI-NOTIFY 2 'Detecting drives, please wait...'
 
 			#Copy FSTAB to ram, faster exe
 			cp /etc/fstab "$FP_TEMP_FSTAB"
 
 			#Get list of all attached drives:
-			MAX_DRIVES=$(blkid -o device | wc -l)
-			local fp_temp='/tmp/.dietpi-drive_manager_devlist'
-			blkid -o device > $fp_temp
-			readarray -t aDRIVE_MOUNT_SOURCE < $fp_temp
+			readarray -t aDRIVE_MOUNT_SOURCE <<< "$(blkid -o device)"
```

Comment:
Use of here strings to feed array, avoid temp file creation, is just read as single time.

Reply:
Nice 👍